### PR TITLE
Mini challenge 2

### PR DESCRIPTION
**Pull request by adrianchavez123**
Adding basic test for second challenge.
Answers:
1 - Using create-react-app, what do we need to set up for testing?

    create-react-app comes with jest library, so you can start creating your test without care about installing or configuring anything.
2 - What components are worth to test in your development?
    
    In my option components that have side effects should be tested.
    For tesing, I mean that a component gets some input and that input changes the output of behaivor. it must be tested.
    For example a component that only renders jsx/html, I don't think we need to test it.

3 - Can you apply TDD once you already created components?
    
     Talking about TDD, its better to create a failing test first, then build the necessary code (we can say create the component) to make the test pass and the second step is to refactor the code.
    When you build the component first, it's possible that tests are not covering all the scenarios because the component is already working and some test cases might be missing.

Example output:
image

### CODE REVIEW

File: public/_redirects [hunk 1]
```diff
@@ -0,0 +1 @@
+/* /index.html 200
```
Comment (PacoMojica) [inline public/_redirects:1]: Good, I see you added the file to solve the `Page not foud` error 👍

File: src/components/App/App.test.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App.component';
+
+it('renders the app', () => {
+  render(<App />);
+  expect(screen.getByText('Welcome to the Challenge!')).toBeInTheDocument();
+});
```
Comment (PacoMojica) [inline src/components/App/App.test.jsx]: You can wrap the test case in with a describe and add a more expressive description to get more information when you run the tests, something like:

```suggestion
describe('App Component', () => {
  it('should render the app', () => {
    render(<App />);
    expect(screen.getByText('Welcome to the Challenge!')).toBeInTheDocument();
  });
});
```

**Review by PacoMojica — COMMENTED** [review]
Hey Adrian, I see you fixed the `page not found` error, the tests look good, in the future you may need to add more test cases to improve the coverage, to get the coverage data run 

```
jest --coverage
```

Great work!

**Comment by adrianchavez123** [comment]
merging...
further mini challenges will need this changes.

**Pull request merged**